src/status_im/subs/wallet/wallet_test.cljs

```diff
@@ -456,6 +464,46 @@
         :tokens                    tokens-0x2})
       (rf/sub [sub-name])))))
 
+(h/deftest-sub :wallet/accounts-with-current-asset
+  [sub-name]
+  (testing "returns the accounts list with the current asset using token-symbol"
+    (swap! rf-db/app-db
+      #(-> %
+           (assoc-in [:wallet :accounts] accounts-with-tokens)
+           (assoc-in [:wallet :ui :send :token-symbol] "ETH")))
+    (let [result (rf/sub [sub-name])]
+      (is (match? result
+                  [{:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
+                    :network-preferences-names #{}
+                    :customization-color       nil}]))))
+
+  (testing "returns the accounts list with the current asset using token"
+    (swap! rf-db/app-db
+      #(-> %
+           (assoc-in [:wallet :accounts] accounts-with-tokens)
+           (assoc-in [:wallet :ui :send :token] {:symbol "ETH"})))
+    (let [result (rf/sub [sub-name])]
+      (is (match? result
```

Comment:
In this repo, the majority of the assertions are `(= expected actual)` or `(match? expected actual)` not the other way around as it's used here. It's just a consistency thing, not that one is way is wrong. Although in the newest version of clj-kondo, it even has a new linter that will check the expected argument appears first by default (https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#equals-expected-position). We can't use this linter version yet.